pkg/corerp/datamodel/environment.go

```diff
@@ -38,11 +38,42 @@ func (e *Environment) ResourceTypeName() string {
 
 // EnvironmentProperties represents the properties of Environment.
 type EnvironmentProperties struct {
-	Compute    rpv1.EnvironmentCompute                           `json:"compute,omitempty"`
-	Recipes    map[string]map[string]EnvironmentRecipeProperties `json:"recipes,omitempty"`
-	Providers  Providers                                         `json:"providers,omitempty"`
-	Extensions []Extension                                       `json:"extensions,omitempty"`
-	Simulated  bool                                              `json:"simulated,omitempty"`
+	Compute      rpv1.EnvironmentCompute                           `json:"compute,omitempty"`
+	Recipes      map[string]map[string]EnvironmentRecipeProperties `json:"recipes,omitempty"`
+	Providers    Providers                                         `json:"providers,omitempty"`
+	RecipeConfig RecipeConfigProperties                            `json:"recipeConfig,omitempty"`
```

Comment:
I am a fan of adding new properties to the end. Would the order matter here?

Reply:
not really but i added before recipes because recipeConfig is usually configured before recipes.